apps/backoffice/src/lib/be/apim-service.ts

```diff
@@ -49,42 +52,38 @@ export const getApimService: () => ApimUtils.ApimService = cache(() => {
   );
 });
 
-const getAxiosInstance = async () => {
+const getAxiosInstance = cache((azureAccessToken: string) => {
```

Comment:
**chore:** now cache is no more needed here

Reply:
done

Reply:
needed in change after review